### DISCUSSION TITLE
Why is GitHub Pulse contributors graph missing?

### DISCUSSION THREAD

**Discussion by doivosevic** [community/community#31200] [Other Feature Feedback, Questions, & Ideas]
So for years there has been the graph as shown https://github.blog/2013-04-18-get-up-to-speed-with-pulse/ . I'm specifically talking about this section [![enter image description here][1]][1]

Last week it dissappeared from all projects, but I cannot find any mention on GitHub changelog or wherever about this happening. Does anyone have more info?

  [1]: https://i.stack.imgur.com/xvVr8.png

**Comment by bjorn** [comment]
Just checked the Pulse on my project. If this graph was ever gone, it appears to have returned.

![image](https://user-images.githubusercontent.com/531764/224752788-029ebbb8-b2ed-4232-917b-39d252917746.png)

**Comment by doivosevic** [comment]
yeah it's been back for a while
